README.md

```diff
@@ -58,12 +58,39 @@ zamp run -i zamp/data/sra-samples.tsv \
 ```
 
 ### Evaluate database
+The module `zamp insilico` allows you to insilico test a pair of PCR primers and evaluate their suitability to correctly amplify and classify taxons of interest. It also allows you  to assess if your database is able to correctly determine the taxonomy of your species of interest, based on the expected amplicon.
+
+To do so, the module performs an insilico PCR with your primer pair on a collection of publicly available assemblies (NCBI). The extracted amplicons are processed and classified by the main zAMP module and compared to the expected NCBI taxonomy. 
+The output is a summary table indicating whether amplification occurs on the species of interest with the specified primer pair, how many amplicons are extracted from your query assemblies, and whether the taxonomy obtained with your database corresponds to the expected.
+As input, you need to provide the taxons to investigate as assembly accession names, tax IDs or queries, as well as the PCR primer pair and your database to evaluate (prepared by `zamp db`).
+
+
+Example usage cases:
+
+1. Using bacteria assembly accession names:
 ```sh
 zamp insilico -i zamp/data/bacteria-accs.txt \
 -db greengenes2 --accession \
 --fw-primer CCTACGGGNGGCWGCAG --rv-primer GACTACHVGGGTATCTAATCC 
 ```
 
+2. Using fungi tax IDs (requires additional ITS amplicon-specific parameters):
+```sh
+zamp insilico -i zamp/data/fungi-taxa.txt \
+-db unite_db_v10 \ 
+--fw-primer CYHRGYYATTTAGAGGWMSTAA --rv-primer RCKDYSTTCWTCRWYGHTGB \
+--minlen 50 --maxlen 900  --amplicon ITS
+```
+
+3. Using a query term. In this example, 100 assemblies will be downloaded per taxon (`nb 100`) including non-reference assemblies (`not-only-ref`):
+```sh
+zamp insilico -i "lactobacillus" \
```

Comment:
Have you tested `zamp insilico -i lactobacillus` ? (without quotes)

Reply:
no